setup.py

```diff
@@ -46,6 +46,7 @@ def get_packages(package):
 minimal_requirements = [
     "click==7.*",
     "h11>=0.8",
+    "h2>=4.0.0",
```

Comment:
I think @tomchristie expressed a wish for this be an optional requirement, but I can't find the issue. 

Just pinging to confirm :kissing: